### PR TITLE
LPS-102933

### PR DESCRIPTION
**Pull request by smailbiala**

### CODE REVIEW

**Comment by AkramYoussoufi** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/MyFormTest.testcase [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+	tearDown {
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "test@liferay.com",
+			userLoginFullName = "Test Test");
+	}
+
+	@description = "Create a New form with Text feild and Numeric feild"
+	@priority = 5
+	test AddFormWithTextFeildAndNumericFeild {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyFormTest.testcase:28]: pay attention .... "Feild"?

File: portal-web/test/functional/com/liferay/portalweb/tests/MyFormTest.testcase [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+			Form.gotoAddField(
+				fieldPositionNumber = 1,
+				fieldType = "Numeric");
+
+			Form.editFieldLabelMetal(fieldValue = "Age");
+
+			Form.save();
+		}
+
+		task ("When The Form Is Filled In Both Lg anguages") {
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyFormTest.testcase:57]: pay attention... "Lg anguages" ?

**Review by manoelcyreno — CHANGES_REQUESTED** [review]

**Comment by manoelcyreno** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/MyFormTest.testcase [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+		}
+	}
+
+	tearDown {
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "test@liferay.com",
+			userLoginFullName = "Test Test");
+	}
+
+	@description = "Create a New form with Text feild and Numeric feild"
```
Comment (manoelcyreno) [inline portal-web/test/functional/com/liferay/portalweb/tests/MyFormTest.testcase:26]: feild? 
pay attention ...

**Comment by manoelcyreno** [comment]
ci:test:sf

**Comment by AkramYoussoufi** [comment]
ci:test:sf

**Comment by manoelcyreno** [comment]
good job, pay more attention on the details in the future ;)

**Comment by manoelcyreno** [comment]
ci:close

**Pull request closed without merging**